inc/saiport.h

```diff
@@ -2284,6 +2331,53 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_OPER_PORT_FEC_MODE,
 
+    /**
+     * @brief Read cable pair state.
+     *
+     * Returns pair states sequentially from list index 0 to n (n = number of pairs - 1)
+     *
+     * @type sai_s32_list_t sai_port_cable_pair_state_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_CABLE_PAIR_STATE,
+
+    /**
+     * @brief Get cable pair length
+     *
+     * Returns cable pair length sequentially from list index 0 to n (n = number of pairs - 1)
+     *
```

Comment:
what does pair mean? what is the unit?

Reply:
roughly how long it takes to run the cable diagnostics. also, for the cable diagnostics to run successfully, do we need support from other side?